### DISCUSSION TITLE
"open you link" generate a blank page. and previous Containers Page [notify_push ] shows a yellow icon,

### DISCUSSION THREAD

**Discussion by tatadudu** [nextcloud/all-in-one#4659] [Questions] [answered]
see its logs below.

Connection to nextcloud-aio-nextcloud (172.19.0.4) 9001 port [tcp/*] succeeded!
The notify_push binary was not found.
Most likely is DNS resolution not working correctly.
You can try to fix this by configuring a DNS server globally in dockers daemon.json.
See https://dockerlabs.collabnix.com/intermediate/networking/Configuring_DNS.html
Afterwards a restart of docker should automatically resolve this.
Additionally, make sure to disable VPN software that might be running on your server
Also check your firewall if it blocks connections to github
If it should still not work afterwards, feel free to create a new thread at https://github.com/nextcloud/all-in-one/discussions/new?category=questions and post the Nextcloud container logs there.

stop all and restart get no help for me.

**Comment by szaimen** [comment, marked as answer]
> Most likely is DNS resolution not working correctly.
> You can try to fix this by configuring a DNS server globally in dockers daemon.json.
> See https://dockerlabs.collabnix.com/intermediate/networking/Configuring_DNS.html
> Afterwards a restart of docker should automatically resolve this.
> Additionally, make sure to disable VPN software that might be running on your server
> Also check your firewall if it blocks connections to github

Hi, did you follow these instructions?

**Comment by tatadudu** [comment]
yes. had followed.


I think I find the root cause.


It was a caddy config issue,&nbsp; and pic2 help me.&nbsp;


I didn't set the&nbsp; localhost to 127.0.0.1


now it is okay and get pic1 by pic3 help or else i cant see nextcloud, but only AIO pages


thanks a lot!



---Original---
From: "Simon ***@***.***&gt;
Date: Fri, May 17, 2024 12:57 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [nextcloud/all-in-one] "open you link" generate a blank page. andprevious Containers Page [notify_push ] shows a yellow icon, (Discussion#4659)




  
Most likely is DNS resolution not working correctly.
 You can try to fix this by configuring a DNS server globally in dockers daemon.json.
 See https://dockerlabs.collabnix.com/intermediate/networking/Configuring_DNS.html
 Afterwards a restart of docker should automatically resolve this.
 Additionally, make sure to disable VPN software that might be running on your server
 Also check your firewall if it blocks connections to github
  
Hi, did you follow these instructions?
